src/docs/patterns/lists.mdx

```diff
@@ -60,7 +197,142 @@ Occasionally, you may want users to be able to re-order the items in a list manu
+
+Drag and drop is a special case for list sorting. It re-orders the items in a list manually. This should be accomplished by placing a drag handle as the leftmost element in the row. The ability to re-order the list should be locked behind an Edit Mode toggle mechanism.
+
+## Local List Actions
+
+<ImageGrid
+    images={[LocalListActionsImage]}
+    caption={[
+        '1: A more_vert icon, used to open a menu for more list actions.',
+        '2: A chevron, usually opens up a new page with list item details.',
+        '3: A few icons that serve as list action buttons.',
+        '4: List actions can be used together.',
+        '5: An inline checkbox to toggle a status.',
+        '6: An inline text field for data entry.',
+    ]}
+    gridImageProps={{ sm: 8 }}
+/>
+
+If there are actions that you would like to perform on individual list items, you may add interactive elements such as icons or a 3-dot action menu. Depending on the space available, you typically will want to use the action menu if you have more than three actions that can be taken.
+
+> If your list item is clickable for the entire item, then you should not arrange any extra local list actions, as the two interactions can intervene with each other.
+
+### Navigating Down a Tree Structure
+
+<ImageGrid
+    images={[NavigateListImage]}
+    caption={['1: A list used for separated pages.', '2: A list displaying a folder structure.']}
+    regularWidth
+/>
+
+Lists can be used to navigate a tree-like structure in an application. See [the navigation pattern](/patterns/navigation).
+
+### Actions on Selected Items
+
+<ImageGrid images={[MultiSelectImage]} regularWidth />
+<DemoCard repository={'multiselect-list'} angular={'https://stackblitz.com/github/pxblue/angular-design-patterns?file=src%2Fapp%2Fpages%2Flist%2Fmultiselect-list%2Fmultiselect-list.component.ts'} react={'https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/lists/multiselect-list/index.tsx&initialpath=multiselect-list'} ionic reactNative />
+
+You may want to take action on several list items at once, but not all. The easiest way to do this is through a multi-select list. You may optionally add a highlight color for the selected items.
+
+## Responsive Design
+
+When it comes to responsive design and progressive disclosure, it can be tricky to fit all the information from a list onto a smaller screen. Here are some best practices on designing for mobile phone:
+
+-   If a list item has too many elements in it and doesn't fit, try converting the item into a summary card, with a "view more" option to view the full data.
+-   Avoid using tables.
+-   Try adding gestures (long-press, swipe-left) that fits with the phone's OS ecosystem.
+
+<Divider />
+
+# Design Specifications
+
+<ImageGrid images={[Spec1Image, Spec2Image, Spec3Image, Spec4Image, Spec5Image, SpecTypographyImage]} regularWidth />
```

Comment:
Specs 1:
Text with Badges? --> Text with Action buttons?

Specs 2:
should we specify that the icon is 24?
Should the spec present the avatar space as 40, the way it's implemented (as you have it), or should we present it as the size of the icon/avatar and the right margin (the way it looks)?

Specs 4:
I don't think the timestamp section is hardcoded with a width. It's just a 32 (i think) margin between leftContent and the title.

Reply:
> Should the spec present the avatar space as 40, the way it's implemented (as you have it), or should we present it as the size of the icon/avatar and the right margin (the way it looks)?

I personally prefer the way it is implemented, bc it puts more rationales for the anatomy

Reply:
> I don't think the timestamp section is hardcoded with a width. It's just a 32 (i think) margin between leftContent and the title.

If it is a margin, then your title & subtitle will shift left & right depends on the timestamp length.

Reply:
In the component it's implemented as a margin, so unless you go in and set a width yourself, then the other content will shift, yes. We can't set a width in the component because the left content can be _anything_, not just a timestamp.

If we are saying 'in the specific case of a timestamp, you should make the timestamp 80dp wide',  I'm fine with that.

Reply:
yeah that is what I'm trying to do here